package-test.yaml

```diff
@@ -6,7 +6,7 @@ tasks:
     name: package
     image: algorand/mule-linux-ubuntu
     version: '{{ docker.Version.outputs.version }}'
-    dockerFilePath: docker/build/mule.ubuntu.Dockerfile
+    dockerFilePath: docker/build/mule.go.debian.Dockerfile
```

Comment:
If we're tagging this as being sourced from Debian, then shouldn't we update the image names from mule-linux-ubuntu in this file?